File: 30.5.19/src/app/models/dto/requestDto.ts
export class RequestDto{
    
    request_id:number;
    request_name:string;
    date_from:Date;
    date_to:Date;
    user_id:number;
    day:number;
    hour:number;  
    day_in_month:number;  
    frequency_id:number;  
    file_stream:number[];
    recording_stream:number[];
     //:BinaryType;////לשאול סוג
    file_name:string;
    file_id;  
    recording_id; 
    is_relevant:Boolean;
    IsAuthorized:Boolean;  
    ErrorMessage:string;  
}

File: 30.5.19/Entities/DTO/RequestDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Entities.DTO
{
    public class RequestDto
    {

        public int request_id { get; set; }
        public string request_name { get; set; }
        public System.DateTime date_from { get; set; }
        public System.DateTime date_to { get; set; }
        public int user_id { get; set; }
        public Nullable<int> day { get; set; }
        public Nullable<int> hour { get; set; }
        public Nullable<int> day_in_month { get; set; }
        public int frequency_id { get; set; }
        public byte[] file_stream { get; set; }
        public byte[] recording_stream { get; set; }
        public string file_name { get; set; }
        public System.Guid file_id { get; set; }
        public System.Guid recording_id { get; set; }
        public bool is_relevant { get; set; }
        public bool IsAuthorized { get; set; }
        public string ErrorMessage { get; set; }
        public string userMail { get; set; }
        // public virtual User1 User { get; set; }

    }
}
